sno/working_copy.py

```diff
@@ -230,6 +232,11 @@ def delete(self):
         # https://www.sqlite.org/shortnames.html
         for f in Path(self.full_path).parent.glob(Path(self.path).name + "-*"):
             f.unlink()
+        try:
```

Comment:
Maybe don't delete this config - see https://github.com/koordinates/sno/issues/192 - this config is now written during `sno init` and so can be present even when the working copy is not. Deleting a working copy shouldn't necessarily clear the config - it depends what is required: should the next `sno checkout` recreate a working copy using the existing config, or if it should create a new working copy using the default config. Check to make sure callers would want the config to be lost, and maybe document, or add an extra arg eg clear_config=False

Reply:
I didn't write this, I think this was a bad merge. will remove